packages/neo-one-smart-contract-lib/src/ownership/Ownable.ts

```diff
@@ -0,0 +1,57 @@
+import { Address, SmartContract, createEventNotifier } from '@neo-one/smart-contract';
+
+const notifyTransferOwnership = createEventNotifier<Address | undefined, Address>('transfer_ownership', 'from', 'to');
```

Comment:
the `from` `Address` can never be `undefined`, no?

Reply:
After modifying `transferOwnership` to use `ownerOrThrow` otherwise, the `mutableOwner` as passed is possibly `undefined`.

Reply:
Answer discussed @ standup, going with "Easy to read".